Luminary099/DAP_INTERFACE_SUBROUTINES.agc

```diff
@@ -156,7 +156,7 @@ ALLCOAST	EXTEND			# SAVE Q FOR RETURN
 		QXCH	RUPTREG2
 # Page 1409
 		TC	STOPRATE	# CLEAR RATE INTERFACE.  RETURN WITH A=0
-		LXCH	EBANK		# AND L=EBANK6.  SAVE CALLER'S EBANK.
+		LXCH	EBANK		#   AND L=EBANK6.  SAVE CALLER'S EBANK.
```

Comment:
I'm not sure about this one - contribution guide makes it sound like we might want 3 spaces, but the scan pretty clearly has the "E" in and "A" lined up

Reply:
Contribution guide is just a generalisation of what the layout is like in most of the situations. It sometimes deviates in alignment like here.